lessons/python/intro-data-analysis/from-spreadsheets-to-pandas-extended.ipynb

```diff
@@ -2107,6 +2124,21 @@
    "source": [
     "It is also possible to get too fancy and accidentally hide important messages in the data. However, the fact that you now have access to several ways to plot your data forces you to consider what is actually important and how you can best communicate that message, rather than always making the same plot without considering its strengths and weaknesses."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Resources to learn more <a class=\"anchor\" id=\"resources-to-learn-more\"></a>\n",
+    "\n",
+    "The documentation for these packages are great resources to learn more.\n",
+    "\n",
+    "- [Introduction to `pandas`](http://pandas.pydata.org/pandas-docs/stable/10min.html)\n",
```

Comment:
https on all these links.

Reply:
Classic! I just installed https everywhere, hopefully that will rid the problem for the future.